OutOfSchool/OutOfSchool.DataAccess/Models/Email.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.Services.Models;
+
+public class Email
+{
+    [DataType(DataType.EmailAddress)]
+    [MaxLength(256)]
```

Comment:
can you move that `256` to `Constants` ?

Reply:
Updated